docs/guides/administrator/adding-and-managing-tenants.md

```diff
@@ -0,0 +1,144 @@
+# Adding and Managing Tenants Guide
+
+This guide will show you how to:
+
+* Add tenants during the initial create operation
+* Add new tenant to a running cluster (with public and private keys)
+* Delete a tenant
+
+## Introduction
+
+Opstrace instances support multiple, secured tenants to logically isolate concerns on otherwise shared infrastructure.
```

Comment:
Prefer singular: "An Opstrace instance supports ..."

Reply:
Leaving out the word "instance" entirely as it is not strictly necessary.